backend-zs/routes/story.routes.js

```diff
@@ -6,6 +6,9 @@ function storyRoutes() {
 
     router.get('/', storiesController.getStories)
     router.get('/:id', storiesController.getStory)
+    router.delete('/:id', storiesController.deleteStory)
+    router.post('/', storiesController.createStory)
```

Comment:
check if the creating user is an admin (middleware already exists in the projects controller)